php/PostTypes/BlockPost.php

```diff
@@ -348,8 +346,13 @@ public function list_table_content( $column, $post_id ) {
 						'wbr'  => [],
 					]
 				);
+			} elseif ( ! empty( $block->template_markup ) ) {
+				esc_html_e( 'Template Editor markup found', 'genesis-custom-blocks' );
+			} else {
+				esc_html_e( 'No template editor markup or template found', 'genesis-custom-blocks' );
```

Comment:
The 'Template' column now shows whether there's markup from the built-in editor:

![editor-here](https://user-images.githubusercontent.com/4063887/118778338-9b28c680-b84f-11eb-832b-d8841dc8575d.png)